packages/block-editor/src/components/block-navigation/utils.js

```diff
@@ -11,3 +16,17 @@ export const getBlockPositionDescription = ( position, siblingCount, level ) =>
 		siblingCount,
 		level
 	);
+
+/**
+ * Returns true if the client ID occurs within the block selection or multi-selection,
+ * or false otherwise.
+ *
+ * @param {string}          clientId              Block client ID.
+ * @param {string|string[]} selectedBlockClientId Selected block client ID, or an array of multi-selected blocks client IDs.
+ *
+ * @return {boolean} Whether the block is in multi-selection set.
+ */
+export const isClientIdSelected = ( clientId, selectedBlockClientId ) =>
```

Comment:
Very minor, but I think it'd be great to make this function's param match the newer `selectedBlockClientIds` naming.